crates/bevy_reflect/src/impls/std.rs

```diff
@@ -41,6 +41,11 @@ impl_reflect_value!(String(Debug, Hash, PartialEq, Serialize, Deserialize));
 impl_reflect_value!(Result<T: Clone + Reflect + 'static, E: Clone + Reflect + 'static>());
 impl_reflect_value!(HashSet<T: Hash + Eq + Clone + Send + Sync + 'static>());
 impl_reflect_value!(Range<T: Clone +  Send + Sync + 'static>());
+impl_reflect_value!(RangeInclusive<T: Clone +  Send + Sync + 'static>());
+impl_reflect_value!(RangeFrom<T: Clone +  Send + Sync + 'static>());
+impl_reflect_value!(RangeTo<T: Clone +  Send + Sync + 'static>());
+impl_reflect_value!(RangeToInclusive<T: Clone +  Send + Sync + 'static>());
```

Comment:
Nit: You have an extra space before each `Send` (likely due to `Range` having one too).
```suggestion
impl_reflect_value!(Range<T: Clone + Send + Sync + 'static>());
impl_reflect_value!(RangeInclusive<T: Clone + Send + Sync + 'static>());
impl_reflect_value!(RangeFrom<T: Clone + Send + Sync + 'static>());
impl_reflect_value!(RangeTo<T: Clone + Send + Sync + 'static>());
impl_reflect_value!(RangeToInclusive<T: Clone + Send + Sync + 'static>());
```

Reply:
I'd make these changes myself, but I've been disallowed from pushing to this branch :)
Worth blocking on as we'll just forget about it if we merge now.